### DISCUSSION TITLE
2021 12 21 tarihli dersdeki  kod

### DISCUSSION THREAD

**Discussion by Ferbman** [19/classroom#1] [Ruby] [answered]
![image](https://user-images.githubusercontent.com/91966086/148570802-11874d47-f05e-414b-838e-4fd6fa6b6b44.png)
buradaki db ne işe yarıyor

**Comment by roktas** [comment, marked as answer]
@KeremKaradenizz metodun dönüş değeri.  O satırı kaldırdığında ne olduğunu bir dene.  Metotlarla ilgili çok temel bir şeyi sormuşsun.  Ders kayıtlarını dikkatli şekilde izlemeni öneririm.

**Reply by Ferbman** [reply]
dönüş değeri belirtmek için return kullanmamız gerekiyor diye biliyordum o yüzden dönüş değeri değil diye düşündüm .
return kullanmadan sona  yazarsak dönüş değeri olarak mı algılıyor.

**Reply by roktas** [reply]
Sorunun cevabı evet.  Bak tekrar yazayım (burada takılmışsan başka pek çok şeyi de atladığından kuşkuluyum).

> Metotlarla ilgili çok temel bir şeyi sormuşsun. Ders kayıtlarını dikkatli şekilde izlemeni öneririm.
